src/components/screens/public/LoginProvider/modules/Provider/specs/Provider.container.spec.js

```diff
@@ -102,14 +102,5 @@ describe('ProviderContainer', () => {
       expect(wrapper.emitted().complete.length).toBe(1);
       expect(wrapper.emitted().complete[0]).toEqual([{ redirect }]);
     });
-
-    it('should emit close', () => {
```

Comment:
We have [event](https://github.com/endpass/endpass-auth/pull/287/commits/8bb72adfd323f330054b0baf76ff5b9eff2c4758#diff-c65d3c0dc75027f117ca7dc0c257cccdR5). Do we need test it?

Reply:
yes, my fault, will revert